### DISCUSSION TITLE
The new feed is spam - Stop showing activity from people I'm not following

### DISCUSSION THREAD

**Discussion by jluttine** [community/community#55865] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

In short: The new (beta?) feed is spam. Spam spam spam. It shows a lot of stuff that I have never asked for and I don't seem to have any control over. It shows that X did something if I've collaborated with X. If I wanted to know when X does something, I'd follow him/her. Don't show anything from people I'm not following. Period. If I care about someone's activity, I'll follow that person. Or why do you have the "follow" feature in the first place? You could perhaps suggest that I'd follow a person if you think I might want to do that..

If I try to choose "Show less activity like this", it offers an option that "I want to see fewer <some activity>". No, that's not what I want. I want to see that activity from the people I have chosen to FOLLOW. But other people, no. I don't want to be spammed. I'll follow someone if I'm interested.

Sorry for the outrage but this is just so annoying. It has made the feed completely useless. I used to like it and use it daily. Now it's useless and I haven't found any way to make it useful again. It's just spam about people I'm not even following.

**Comment by Olf0** [comment]
:+1: Absolutely correct!

@GitHub, please stop this immediately!

**Comment by roncanepa** [comment]
I just came here to say the same exact thing.  This is ridiculous.

**Comment by jamesmacintyre** [comment]
Agree

**Comment by jluttine** [comment]
Oh, well, I just now noticed that the old feed is in another tab with title "Following". Nice, it's still available! I hope Github is not planning to remove that. But still the new feed is spam. At least try to find something interesting, not just show every minor action from everyone who has ever pushed commits to a same repository as I. That's a loooot of people if one has contributed to popular open source projects..

**Comment by sirkon** [comment]
<img width="712" alt="image" src="https://github.com/community/community/assets/56399/3eae8355-6fe6-424f-8790-eade6c5c9f29">

Just use the filter. And switch to the "Following", cause this "For you" is just a utterly useless gimmick.

**Comment by lucthienphong1120** [comment]
agree

**Comment by adriank** [comment]
Don't turn GitHub into LinkedIn! LinkedIn was a great professional site, now it's unbearable. GitHub's feed will turn into the same spam machine as in LIn. Thankfully, there are other platforms to store git repos.

**Comment by bionade24** [comment]
Please at least make an option to let the old following be the default again. "For you" is useless Spam. What financial benifit does it even have? Do you plan using it for advertising?

**Comment by myklemykle** [comment]
Super not-interested in yet another useful tool trying to turn itself into yet another addictive drip of internet drama.  Is that really the best MS can come up with for an institution like GitHub?

**Comment by otom8-io** [comment]
It remembers me of why I stopped using Facebook. I don't care what you care for me, don't bother me with the activity of people I don't even know. Please remove this. The home page should only show me the projects I am working on.

**Comment by manurocck** [comment]
> If I try to choose "Show less activity like this", it offers an option that "I want to see fewer ". No, that's not what I want. I want to see that activity from the people I have chosen to FOLLOW. But other people, no. I don't want to be spammed. I'll follow someone if I'm interested.

There's an explorer extension to erradicate the **news feed** called **_"News Feed Erradicator"_**
It doesn't only work in GitHub but also does the same in Facebook, Instagram, LinkedIn (etc.)

You can use it so you don't get distracted over these catchy feeds and to control a little more what you see...

It might not be a direct solution to your problem but you can avoid it - at least.


> ![image](https://github.com/community/community/assets/22857096/383fc12b-6b48-436b-890e-a75a6f50b240)

**Comment by psa** [comment]
Same. All I want to be greeted with in most cases is a list of my repositories. Because, chances are, I'm here to do something on one of them and the faster I can get to it the better.

**Comment by JC3** [comment]
One solution I found is to make github.com/yourusername your go-to page instead of just plain github.com.
